e2e/utils.go

```diff
@@ -1083,7 +1087,8 @@ func validateController(f *framework.Framework, pvcPath, appPath, scPath string)
 	if err != nil {
 		return fmt.Errorf("failed to delete storageclass: %w", err)
 	}
-	err = createRBDStorageClass(f.ClientSet, f, defaultSCName, nil, nil, deletePolicy)
+	err = createRBDStorageClass(f.ClientSet, f, defaultSCName, scOptions, scParams,
```

Comment:
Validate pvc encryption before and after binding pvc to application pod.

Reply:
@Madhu-1 ,Can you be more specific about what to validate in the before binding pvc to application pod case ?

Reply:
`validateController` doesn't do any validation related to encryption, Now with This PR mirroring support for encryption is added please add validation to check mountpoint is encrypted.